### DISCUSSION TITLE
What's the difference between position and positionAbsolute?

### DISCUSSION THREAD

**Discussion by shemayas** [xyflow/xyflow#2635] [Q&A] [answered]
I'm implementing what @moklick wrote [here](https://github.com/wbkd/react-flow/issues/2440#issuecomment-1273010975) by changing the node position on node change and I'm not sure what is the difference between `position` and `positionAbsolute` in the `NodePositionChange`.

Should I update both of them?

**Comment by bcakmakoglu** [comment, marked as answer]
You probably don't need to touch `positionAbsolute`
`positionAbsolute` is mostly relevant for flows with sub-nodes, i.e. parent-child nodes.
As the name suggests, it is the *absolute* position of a node, while `position` is the relative position of a child node in relation to it's parent.

So `{ x: 0, y: 0 }` of a child node means, 0, 0 in relation to the parent, *not* in relation to the whole canvas.
If you're not using child-nodes, it's the same, there's no difference in what the absolute position and relative position would be.

**Reply by shemayas** [reply]
Thanks
